org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/ast/Bounds.java

```diff
@@ -0,0 +1,99 @@
+package edu.mit.csail.sdg.ast;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import edu.mit.csail.sdg.alloy4.ConstList;
+import edu.mit.csail.sdg.alloy4.Err;
+import edu.mit.csail.sdg.alloy4.ErrorWarning;
+import edu.mit.csail.sdg.alloy4.Pos;
+
```

Comment:
let's add some doc comments for public classes/fields

Reply:
added here and other files.